src/extension/handler/uri.ts

```diff
@@ -103,13 +105,25 @@ export class RunmeUriHandler implements UriHandler {
         fileToOpen: string
     ) {
         progress.report({ increment: 0, message: 'Cloning repository...' })
-        const terminal = window.createTerminal(`Runme Terminal #${NEXT_TERM_ID++}`, '/bin/sh')
+        const terminal = window.createTerminal(`Runme Terminal #${NEXT_TERM_ID++}`)
         terminal.show(true)
 
-        terminal.sendText(`git clone ${repository} ${targetDirUri.fsPath}`)
+        terminal.sendText(`git clone ${gitSSHUrlToHTTPS(repository)} ${targetDirUri.fsPath}`)
```

Comment:
Not 100% sure, however, I'm not seeing the HTTPS URL being checked for validity to prevent injections here.

Reply:
There are two places:
- input parsing to prevent injection: [`parseParams`](https://github.com/stateful/vscode-runme/blob/ac53278c1041d6afb29c2c4d95648a50ca295fab/src/extension/handler/utils.ts#L177-L198) (which is called right at the beginning of the uri handler
- validating format of passed in url: [`getSuggestedProjectName`](https://github.com/stateful/vscode-runme/blob/ac53278c1041d6afb29c2c4d95648a50ca295fab/src/extension/handler/utils.ts#L138-L158)